Dockerfile

```diff
@@ -36,7 +36,7 @@ ARG hypopg_release=1.3.1
 ARG pgvector_release=0.4.0
 ARG pg_tle_release=1.3.2
 ARG index_advisor_release=0.2.0
-ARG supautils_release=2.2.0
+ARG supautils_release=2.2.1
```

Comment:
I'd prefer upgrading of packaged be in a separate PR with nix things

Reply:
@pashkinelfe I needed to test these upgraded versions in staging.

Reply:
Unless it doesn't affect current/future AMI builds (I suppose it does), adding these changes in a separate PR is much better.